native/src/lib.rs

```diff
@@ -297,8 +314,8 @@ fn bbs_sign(mut cx: FunctionContext) -> JsResult<JsArrayBuffer> {
     let message_bytes = obj_field_to_vec!(&mut cx, js_obj, "messages");
 
     let mut messages = Vec::new();
-    for i in 0..message_bytes.len() {
-        let message = obj_field_to_field_elem!(&mut cx, message_bytes[i]);
+    for message_byte in message_bytes {
+        let message = obj_field_to_field_elem!(&mut cx, message_byte);
         messages.push(message);
     }
```

Comment:
I wonder if in future we could be more idiomatic. Can't remember if mutable borrow allowed in map closure 😅 

```rust
let messages: Vec<_> = message_bytes
  .iter()
  .map(|b| obj_field_to_field_elem!(&mut cx, message_byte))
  .collect();
```

Reply:
Yeah I thought about making the entire codebase more functional, but I left it procedural since it's how the project is developed